### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by mirouhml**
- Initialise React app.
- The building blocks of are set as re-usable components. Create a directory for the components.
- Add React Router and set two <Route>s and <Link>s for the app's navigation:
  - Books - the default view:
    - It displays the list of books with the Remove button (no funcionality yet).
      - Create a component called Book for displaying a single book and reuse it in a component that displays a list of books.
    - Should have a form for adding a book (no functionality yet).
      - Create a separate component for this form.
- Categories:
  - Should display a button "Check status" only.
- Add basic styling, because it is not required at this point.

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <form>
+      <input type="text" placeholder="Book title" />
+      <select id="categories-dropdown" name="categories" defaultValue="Categories">
+        <option value="categories" disabled>Categories</option>
+        <option value="action">Action</option>
+        <option value="science fiction">Science Fiction</option>
+        <option value="fantasy">Fantasy</option>
+      </select>
+      <button type="button">ADD BOOK</button>
+    </form>
```
Comment (Meltrust) [inline src/components/Form.js]: According to the project requirements:

image

Therefore:

- [ ] Please, include an input for the author of the books.

## Explanation

Please, don't get confused by the sneak peek.  The image that is included there is just meant for the styling.  The form must include the ability to input the author.  Also, in the next milestones, you will need this input.

## Hints

You could shorten the "Book title" input, and in that space insert your "Author" input, next to it.  Like this:

image

That way, we will meet this requirement, and be prepared for the next milestones 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @mirouhml,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- The project renders correctly✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (mirouhml) [reply src/components/Form.js]: Thanks for catching that @Meltrust , I totally forgot about it about it!

**Review by Meltrust — APPROVED** [review]
Hi @mirouhml,


## Wow, you did it 🎉
Brilliant
## Thank you for including the author input, as requested in the previous review💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mirouhml** [comment]
Thanks for the review @Meltrust !

**Pull request merged**